src/mds/CInode.cc

```diff
@@ -2044,9 +2044,10 @@ void CInode::decode_lock_inest(bufferlist::const_iterator& p)
 
 void CInode::encode_lock_ixattr(bufferlist& bl)
 {
-  ENCODE_START(1, 1, bl);
+  ENCODE_START(2, 1, bl);
   encode(get_inode()->version, bl);
   encode(get_inode()->ctime, bl);
+  encode(get_inode()->xattr_version, bl);
```

Comment:
Nice catch @gregsfortytwo!

@lxbsz This needs to be at the tail end...

Reply:
Yeah, @gregsfortytwo is right. I will fix it.
Thanks!